src/App.js

```diff
@@ -1,18 +1,63 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import uuid from 'uuid';
+import Swipeable from './Swipeable.js';
+import Interactions from './Interactions.js';
+
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: 'unknown@mit.edu',
+      workshopCode: 'foo',
+      sessionId: uuid.v4()
+    };
```

Comment:
These are variables that the user will enter on the first screens, and these are placeholder for now.